server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -87,6 +93,24 @@ public void runServer() throws Exception {
     }
   }
 
+  @Override
+  public void registerMetrics(MeterRegistry registry) {
+    lowMemoryMetricGuage =
+        Gauge
+            .builder(METRICS_APP_PREFIX + applicationName + "." + hostname + "."
+                + METRICS_APP_LOW_MEMORY, this, this::lowMemDetected)
+            .description(
+                "reports 1 when process memory usage is above threshold, 0 when memory is okay") // optional
+            .register(registry);
+  }
+
+  private int lowMemDetected(AbstractServer abstractServer) {
+    if (abstractServer.context.getLowMemoryDetector().isRunningLowOnMemory()) {
+      return 1;
+    }
+    return 0;
```

Comment:
```suggestion
    return abstractServer.context.getLowMemoryDetector().isRunningLowOnMemory() ? 1 : 0;
```

Or, this looks cleaner, because it seems to be passing the current object to itself:

```suggestion
    return context.getLowMemoryDetector().isRunningLowOnMemory() ? 1 : 0;
```